.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    open-pull-requests-limit: 6
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "bot: dependencies update"
+    reviewers:
+      - "wordpress-mobile/android-developers"
+    ignore:
+      # The Android Gradle Plugin is a dependency that needs to be in sync with other
+      # in-house libraries due to compatibility with composite build.
+      - dependency-name: "com.android.tools.build:gradle"
+      # Bumping 1.2.1 to 1.3.0 causes some issues, fist spotted in Reader. For more details, see
+      # https://github.com/wordpress-mobile/WordPress-Android/pull/14431
+      - dependency-name: "com.google.android.material:material"
```

Comment:
I wouldn't recommend to ignore it - I believe sooner or later the bump will be necessary. Maybe it's worth to revisit and fix the issue?

Reply:
Thanksor for the review @wzieba ! 🙏 

> I wouldn't recommend to ignore it - I believe sooner or later the bump will be necessary.

Yes, you are right. I just chose to ignore it for now and deal with it explicitly because last time this simple update happened, noone noticed it and it almost when live until we noticed that some layouts are having problems. Thus, I felt that if we don't ignore it for now, when the bot will generate the PR, someone will smoke test it and potentially approve it without doing a thorough test or knowing this wierd correlation between `Material` and `Constraint Layout`.

> Maybe it's worth to revisit and fix the issue?

What if I creat a specific `Material` update issue explaining all about the problems coming with it, that `Constraint Layout` is related somehow and might also need to be updated as well. Then, I can keep this ignoring rule and add a new comment line with the linked issue that should address it. Wdyt? 🤔

Reply:
Personally I'm ok with you creating a specific issue explaining all about the problem, as long as that issue does not end up in limbo and gets forgotten (maybe assigning it a Milestone to target it in a specific future sprint?)

I'd then recommend adding an additional comment on this line to point to said future issue.

Reply:
On it! 💯

Reply:
FYI: I created this accompanying #16132 issue for that matter, let me know what you think. 🙏 

PS: I also added an [extra link comment commit](https://github.com/wordpress-mobile/WordPress-Android/pull/16092/commits/51bf728501f1890ef8c8a0c448e76db73658068b) for that matter.

Reply:
Thanks @ParaskP7 ! I'm going to merge this so it lands before I start the 19.5 code freeze 🎉 
(We can always follow up with an additional PR if @wzieba still disagrees with it being ignored anyway and we decide to change it)

Reply:
Great, thank you so much @AliSoftware ! 🚀 🟢

Reply:
It's all good on my end, thanks for establishing strategy for handling this dependency!